homeassistant/components/cover/zwave.py

```diff
@@ -138,3 +158,12 @@ def device_class(self):
     def supported_features(self):
         """Flag supported features."""
         return SUPPORT_GARAGE
+
+    @property
+    def device_state_attributes(self):
```

Comment:
This is handled in `CoverDevice::state_attributes`, so it's not necessary to include this here, correct?